packages/emoji-mart/src/components/Picker/Picker.js

```diff
@@ -926,6 +926,23 @@ export default class Picker extends Component {
     )
   }
 
+  renderSkinToneIcon(skin) {
+    if (this.props.skinToneEmoji) {
+      return (
+        <Emoji
+          id={this.props.skinToneEmoji}
+          set={this.props.set}
+          size={this.props.emojiSize}
+          skin={skin}
+          spritesheet={true}
+          getSpritesheetURL={this.props.getSpritesheetURL}
+        />
```

Comment:
The emoji is changing position when active:

<img width="127" alt="Screenshot 2022-11-06 at 14 20 09" src="https://user-images.githubusercontent.com/436043/200190552-5697fba6-813e-49c7-b7b9-5f9abb21e683.gif">